CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Contributing
+
+Contributions are welcome from anyone.
+
+To report bugs or issues, open [new issues](https://github.com/brndnmtthws/conky/issues/new).
+To submit code changes, open [new pull requests](https://github.com/brndnmtthws/conky/compare).
```

Comment:
(imho) This section is excessive. We could not mention this part.

Reply:
I trimmed this down.